### PR TITLE
More Python f-strings with flynt

### PR DESCRIPTION
**Pull request by cclauss**
This PR was created with the command: [`flynt .`](https://pypi.org/project/flynt)
```
Running flynt v.0.76

Flynt run has finished. Stats:

Execution time:                            1.944s
Files checked:                             204
Files modified:                            39
Character count reduction:                 592 (0.02%)

Per expression type:
Old style (`%`) expressions attempted:     120/241 (49.8%)
`.format(...)` calls attempted:            19/19 (100.0%)
No concatenations attempted.
F-string expressions created:              119
```

### CODE REVIEW

File: buildconfig/config_emsdk.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def __init__(self, name, envname, exename, minver, defaultlibs, version_flag="--
 
 
         except (ValueError, TypeError):
-            print ('WARNING: "%s" failed!' % command)
+            print (f'WARNING: "{command}" failed!')
```
Comment (novialriptide) [inline buildconfig/config_emsdk.py:81]: Maybe someone in the future should create a PR fixing all of the `print ()`s to `print()`

Comment (cclauss) [reply buildconfig/config_emsdk.py:81]: Yes.   Separate PR.

**Review by dr0id — APPROVED** [review]

**Review by MyreMylar — APPROVED** [review]
LGTM 👍

File: examples/scaletest.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def main(imagefile, convert_alpha=False, run_speed_test=False):
 
 
 def SpeedTest(image):
-    print("\nImage Scaling Speed Test - Image Size %s\n" % str(image.get_size()))
+    print(f"\nImage Scaling Speed Test - Image Size {str(image.get_size())}\n")
```
Comment (illume) [inline examples/scaletest.py:94]: Not that it matters really, but `str()` isn't necessary here.

**Review by illume — APPROVED** [review]
👍 thanks

**Pull request merged**